tools/repo_docs.sh

```diff
@@ -0,0 +1,15 @@
+cd $(dirname $(realpath $0))/witx
+cargo run -p witx-cli -- docs ../../phases/snapshot/witx/wasi_snapshot_preview1.witx --output ../../phases/snapshot/docs.md
```

Comment:
Could this be called from CI to avoid duplicating it?

Reply:
Good point. I'll add an argument to pass in --check on CI